test/packages/parallel/gcp/data_stream/compute/_dev/deploy/tf/env.yml

```diff
@@ -0,0 +1,7 @@
+version: '2.3'
+services:
+  terraform:
+    environment:
+      # pass project id to Terraform
+      - TF_VAR_gcp_project_id=${GCP_PROJECT_ID}
```

Comment:
Shouldn't we preserve casing consistency (upper-case)?

Reply:
This format is consistent with Terraform [documentation about setting input variables from environment variables](https://www.terraform.io/language/values/variables#environment-variables).

I think this consistency helps Terraform practitioners (both beginners and experts alike) more.

I also know we are using `TEST_RUN_ID`, something I think should be changed for the same reason, but I didn't want to address that in this PR.

Reply:
My bad looks like it's aligned with the official docs. Probably it's something we can mention in our docs. Anyway, nothing to adjusted.